app/config/Config.scala

```diff
@@ -69,7 +69,7 @@ class Config(playConfig: Configuration) extends AwsInstanceTags with Logging {
   lazy val presenceUrl: String = s"wss://presence.$domain/socket"
   lazy val presenceClientLib: String = s"https://presence.$domain/client/1/lib.js"
 
-  lazy val preferencesUrl: String = s"https://preferences.${stage.appDomain}/preferences"
+  lazy val preferencesUrl: String = s"https://preferences.$domain/preferences"
```

Comment:
It isn't immediately obvious from the diff that this is a noop.  But since you are changing this, why not also change line 74?

Reply:
I've not made it any more obvious that it's a noop, but I have made it consistent across this whole file now (i.e. used `$domain` everywhere as more succinct